include/SFML/Graphics/Rect.hpp

```diff
@@ -44,94 +44,152 @@ class Rect
 {
 public:
     ////////////////////////////////////////////////////////////
-    /// \brief Default constructor
+    /// \brief Construct the rectangle from position and size.
     ///
-    /// Creates an empty rectangle (it is equivalent to calling
-    /// Rect(0, 0, 0, 0)).
+    /// \param position Position of the rectangle
+    /// \param size     Size of the rectangle
     ///
     ////////////////////////////////////////////////////////////
-    constexpr Rect();
+    constexpr Rect(Vector2<T> rectanglePosition = Vector2<T>(), Vector2<T> rectangleSize = Vector2<T>());
```

Comment:
I don't think default parameters are the way to go; it's not meaningful to have `Rect(position)`, let alone implicit. Either the user should specify both position and size, or none.

Reply:
Agreed. Done

Reply:
Done